Documentation/BuildingWebRTC.md

```diff
@@ -69,8 +66,7 @@ The build artifacts will be located in `~/src/build_webrtc/build/ios/`.
 
 ### Android
 
-**NOTE**: WebRTC for Android can only be built on Linux at the moment.
-
+**NOTE**: WebRTC for Android can only be built on Linux at the moment.  
```

Comment:
let's remove "at the moment" since I don't see that cchanging maybe ever. That's a WebRTC limitation.

Reply:
Adjusted in the next commit.